charts/apisix/templates/service-admin.yaml

```diff
@@ -25,6 +25,7 @@ metadata:
     {{- end }}
   labels:
     {{- include "apisix.labels" . | nindent 4 }}
+    app.kubernetes.io/component: apisix-admin
```

Comment:
In fact, from the semantics of `component`, they are the same componet.
I don’t think this label is reasonable enough.

Reply:
How about `app.kubernetes.io/service` ?

Reply:
👌